crest/Assets/Crest/Crest/Scripts/LodData/OceanDepthCache.cs

```diff
@@ -437,8 +437,8 @@ public bool Validate(OceanRenderer ocean, ValidatedHelper.ShowMessage showMessag
                 {
                     showMessage
                     (
-                        "Depth cache is outdated. Click <i>Populate Cache</i> or re-bake the cache to bring the cache " +
-                        "up-to-date with component changes.",
+                        "Depth cache is outdated.",
+                        "Click Populate Cache or re-bake the cache to bring the cache up-to-date with component changes.",
```

Comment:
sadly, the tooltip does not support it and displays its as `<i></i>` :/.

Reply:
i did consider perhaps using single quotes instead like 'this'

Reply:
Is stripping the tags an option? We would only need to strip it before adding it to the tooltip since the Undo.RecordObject takes the tooltip too.

Reply:
I think a regex (untested) like so will handle all tags: `@"<[^<>]*>"`?

Reply:
yeah that pretty much works. its not too hard to make it fail, but i think its a rabbit hole to make it really solid, and failures are noncritical i would say.

`condition is <b>cat < bat && dog > frog </b>.`

Reply:
True. It's pretty naive, but I didn't expect us to use arrows in such a way. We could change to `<[^<>\s]*>` to handle spaces. I think that should cover quite a lot of cases (probably all of ours).

Reply:
Here is documentation for rich text for reference: https://docs.unity3d.com/Packages/com.unity.ugui@1.0/manual/StyledText.html

If we really needed to we could cover everything since the amount of tags are pretty small. But can leave it until we need it.

Reply:
I'll resolve this since it covers the issue.